webservices/filters.py

```diff
@@ -5,18 +5,42 @@
 from webservices.common import models
 
 
+def is_exclude_arg(arg):
+    # Handle string and int excludes
+    return str(arg).startswith('-')
```

Comment:
I think this is an appropriate abstraction here - if we ever change how we want to handle URL parameters for filtering, particularly around exclusion, we just have to worry about this spot. :-)